pkg/VSConfigFinder/VSConfigFinder.nuspec

```
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>VSConfigFinder</id>
+    <version>$Version$</version>
+    <title>Visual Studio Configuration Finder</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <iconUrl>https://go.microsoft.com/fwlink/?linkid=839265</iconUrl>
+    <licenseUrl>https://github.com/Microsoft/VSConfigFinder/tree/$CommitId$/LICENSE.txt</licenseUrl>
+    <projectUrl>https://github.com/Microsoft/VSConfigFinder</projectUrl>
+    <description>Locate Visual Studio installation configuration files (.vsconfigs) downstream recursively and merge them altogether</description>
+    <summary>Locate Visual Studio installation configuration files (.vsconfigs) downstream recursively and merge them altogether</summary>
```

nit: I think this should say `all together`

```suggestion
    <summary>Locate Visual Studio installation configuration files (.vsconfigs) downstream recursively and merge them all together</summary>
```